doc/packages/ibus.section.md

```diff
@@ -11,7 +11,8 @@ On NixOS, you need to explicitly enable `ibus` with given engines before customi
 ```nix
 { pkgs, ... }: {
   i18n.inputMethod = {
-    enabled = "ibus";
```

Comment:
I've intentionally left the other test (`nixos/tests/fcitx5/default.nix`, not present in this review) using `enabled` to have a test for the backwards compatibility of this change

Reply:
Whoops, this is the docs file. I meant to add this to nixos/tests/installed-tests/ibus.nix